ibm/mas_devops/roles/ocp_setup_mas_deps/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
     - openshift_config.resources is defined
     - openshift_config.resources | length > 0
   set_fact:
-    ocp_install_invoker: "{{ openshift_config.resources[0].data.invoker }}" # Known values: "ROKS"
+    ocp_install_invoker: "{{ openshift_config.resources[0].data.invoker }}" # Known values: "ROKS" or "user" (quickburn)
```

Comment:
The addition to the comment is incorrect (unless we have new evidence that sometimes Fyre uses `openshift-install` instead of `openshift-install-manifest`) ... in this scenario only ROKS is possible, From our testing Fryre doesn't use this configmap, it uses the `openshift-install-manifests` configmap that we check if this one doesn't exist.  see snippet below where we comment on the known possible values in the alternative configmap.  However it may be mroe complicate than that .. e.g. a specific version of OCP on Fyre uses different configmaps perhaps?  What version of OCP on Fyre did we find this configmap being used on that triggered this update to the doc?

```
# If we didn't already set the new facts then it means we couldn't find the "openshift-install" configmap
- name: Get OCP Install Information (openshift-install-manifests)
  when: ocp_install_invoker is not defined
  community.kubernetes.k8s_info:
    api_version: v1
    kind: ConfigMap
    name: openshift-install-manifests
    namespace: openshift-config
  register: openshift_config
- name: Register OCP install configuration variables (openshift-install-manifests)
  when:
    - ocp_install_invoker is not defined
    - openshift_config.resources is defined
    - openshift_config.resources | length > 0
  set_fact:
    ocp_install_invoker: "{{ openshift_config.resources[0].data.invoker }}" # Known values: "user" (in Fyre)
    ocp_install_version: "{{ openshift_config.resources[0].data.version }}"
ocp_install_release: "{{ openshift_config.resources[0].data.version | regex_search('[^.]*.[^.]*') }}"
```

Reply:
You are right @durera ... I got the config map from Jean (who had a fyre cluster), but did not notice the name of the config-map was different (checking again now, and I confirmed it was -manifest one). And now it makes sense your change to include this new config map checking. I will remove this piece of comment!

Reply:
Fixed